### PR TITLE
Add scroll up button to website

### LINKED ISSUE #9108: scroll-up button
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

While I was going through the website, I realized there might be a need for a scroll0-up button on the overview, tutorials, download, and blog pages

### Suggested solution

I will add a green scroll-up button on these pages which will help to enhance the user experience by 3%

### Alternatives considered

_No response_

### PR DESCRIPTION
**Pull request by simondriesen**
Closes #9108

Since the other pull request has been stuck for some time I decided to give it a go as well.

### CODE REVIEW

File: docs/docs/css/main.css [hunk 1]
```diff
@@ -142,11 +142,55 @@ h4 {
...
 
+#back-to-top {
+	position: fixed;
+	right: 1.5em;
+	bottom: 1.5em;
+	width: 46px;
+	height: 46px;
+	border: none;
+	border-radius: 50%;
+  background-color: #eee;
```
Comment (Falke-Design) [inline docs/docs/css/main.css:159]:
```suggestion
	background-color: #eee;
```

**Review by Falke-Design — CHANGES_REQUESTED** [review]
@simondriesen really nice, thank you! What do you think about showing the box-shadow always?

File: docs/docs/css/main.css [hunk 2]
```diff
@@ -1006,6 +1050,11 @@ tr:target {
 		-moz-column-count: 2;
 		column-count: 2;
 	}
+
+	#back-to-top {
+		z-index: 2;
+		box-shadow: rgba(14, 15, 15, 0.3) 0px 2px 10px 0px;
```
[the remark this answers is not in the document]
Comment (simondriesen) [reply docs/docs/css/main.css:1056]: It was a bit of an afterthought to make it pop out more on mobile but I guess it doesn't hurt to have it on desktop too.

**Comment by simondriesen** [comment]
image

I also gave the button a higher z-index because it the "overview" page it was hiding behind the map on mobile.

**Review by Falke-Design — APPROVED** [review]

**Comment by Falke-Design** [comment]
@simondriesen thank you!

**Pull request merged**